composer.json

```diff
@@ -28,14 +28,14 @@
     "require":
     {
         "php":                  "^5.5|^7.0",
-        "psr/cache":            "1.0.0",
-        "cache/adapter-common": "^0.1",
-        "cache/taggable-cache": "^0.2"
+        "psr/cache":            "~1.0",
+        "cache/adapter-common": "^0.2",
```

Comment:
The BC braking feature in 0.2 is that fetchObjectFormCache must return an array. You should do like this: 

```php
protected function fetchObjectFromCache($key)
    {
        return [false, null];
    }
```